suppressor.go

```diff
@@ -23,7 +23,7 @@ var (
 	token    = os.Getenv("SUPPRESSOR_TOKEN")
 	wordlist = []string{"502", "bad gateway", "(?:is\\s+)?(?:(?:\\s+)?the\\s+)?(?:sponsor(?:\\s+)?block|sb|server(?:s)?|api)\\s+(?:down|dead|die(?:d)?)", "overloaded", "(?:sponsor(?:\\s+)?block|sb|server(?:s)?|api) crash(?:ed)?",
 		"(?:(?:issue|problem)(?:s)?\\s+)(?:with\\s+)?(?:the\\s+)?(?:sponsor(?:\\s+)?block|sb|server(?:s)?|api)", "exclamation mark", "segments\\s+are\\s+(?:not\\s+)?(?:showing|loading)",
-		"(?:can't|cannot) submit"}
+		"(?:can't|cannot) submit", "403", "can\S+ (?: \S+)? (?:submit)|(?:see)"}
```

Comment:
I feel like receiving 403 doesn't actually have to do anything with the server being down. I think this should be fine for the time being:
```suggestion
		"(?:can't|cannot) (?:submit|see)"}
```